UnityProject/Assets/Scripts/Items/ItemAttributesV2.cs

```diff
@@ -227,7 +227,7 @@ public IEnumerable<ItemTrait> GetTraits()
 		/// <param name="itemTrait"></param>
 		public bool HasTrait(ItemTrait toCheck)
 		{
-			return traits.Contains(toCheck);
+			return InitialTraits.Contains(toCheck);
```

Comment:
if traits change during gameplay, this wont return the correct traits.
create another separate function for initaltraits

Reply:
Oh yeah was populating sis in the editor and forgot to change it back